plugin/advisor/mysql/advisor_index_type_no_blob_test.go

```diff
@@ -0,0 +1,198 @@
+package mysql
+
+// Framework code is generated by the generator.
+
+import (
+	"testing"
+
+	"github.com/bytebase/bytebase/plugin/advisor"
+)
+
+func TestIndexTypeNoBlob(t *testing.T) {
+	tests := []advisor.TestCase{
+		// CREATE TABLE TABLE OPTION INDEX
```

Comment:
Add some tests for CREATE INDEX statement, please

Reply:
Done.